js/arena.js

```diff
@@ -0,0 +1,30 @@
+ export default class Arena {
+     constructor(name) {
+
+         this.name = name.charAt(0).toUpperCase() + name.slice(1)
+         this.gladiators = []
+     }
+     addGladiator(gladiator) {
+         if (this.gladiators.length == 2) {
+             return console.log('Max Number of Gladiators in Arena ')
+         } else this.gladiators.push(gladiator)
+     }
+     fight() {
+         if (this.gladiators.length == 2) {
```

Comment:
you need to remove losing or eliminated gladiators from the "arena", the gladiators array.